services/ui-src/src/components/pages/ReviewSubmit/McparReviewSubmitPage.tsx

```diff
@@ -11,34 +11,39 @@ import {
   Text,
   useDisclosure,
 } from "@chakra-ui/react";
-import { Modal, ReportContext } from "components";
+import { Modal, ReportContext, StatusTable } from "components";
 // types
-import { ReportStatus } from "types";
+import { AlertTypes, ReportStatus } from "types";
 // utils
-import { useUser, utcDateToReadableDate, convertDateUtcToEt } from "utils";
+import { useUser, utcDateToReadableDate } from "utils";
 // verbiage
 import reviewVerbiage from "verbiage/pages/mcpar/mcpar-review-and-submit";
 // assets
 import checkIcon from "assets/icons/icon_check_circle.png";
 import printIcon from "assets/icons/icon_print.png";
+import { Alert } from "components/alerts/Alert";
 
 export const McparReviewSubmitPage = () => {
-  const { report, fetchReport, updateReport } = useContext(ReportContext);
+  const { report, fetchReport, submitReport } = useContext(ReportContext);
   const { isOpen, onOpen, onClose } = useDisclosure();
 
   const [submitting, setSubmitting] = useState<boolean>(false);
+  const [hasError, setHasError] = useState<boolean>(false);
 
   // get user information
-  const { email, full_name, state, userIsStateUser, userIsStateRep } =
-    useUser().user ?? {};
+  const { state, userIsStateUser, userIsStateRep } = useUser().user ?? {};
 
-  const isPermittedToSubmit = userIsStateUser || userIsStateRep;
+  const isPermittedToSubmit =
+    (userIsStateUser || userIsStateRep) &&
+    report?.status !== "Not started" &&
+    !hasError;
```

Comment:
Question: Are users allowed to submit if the report is in "Submitted" state? Maybe this check should be `report?.status === "In progress"`. We have an [enum for that here](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/e12110fa9ab9e2ad9f66d87e15b532816949c46e/services/ui-src/src/types/index.ts#L181-L185) which is used in other files (though I guess not in our test files for some reason 🤷).

Reply:
Marking 'unresolved' because still pending discussion.

Reply:
resolving under MDCT-2336